rootfs/api/models/config.py

```diff
@@ -137,13 +111,12 @@ def save(self, **kwargs):
                         # error if unsetting non-existing key
                         if key not in data:
                             raise UnprocessableEntity('{} does not exist under {}'.format(key, attr))  # noqa
-
                         data.pop(key)
                     else:
                         data[key] = value
                 setattr(self, attr, data)
 
-            self.set_healthchecks()
+            self._migrate_legacy_healthcheck()
```

Comment:
Yeah, I think we may have to keep it to the next major release given how strict we are usually on breakages. @slack thoughts on how long we should keep it deprecated before removal?